client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
+  const userFavoriteCitiesUrl = "/api/users/favoriteCities";
+  useEffect(() => {
+    async function getData(){
+      try {
+        const result1 = await fetch(userFavoriteCitiesUrl, {
```

Comment:
`result1` and `result2` are not very descriptive variable names. Something like `favoriteCitiesResponse` and `placesResponse` are more clear